commit 5791f5c2fd91efa1f477755654922092931c56d1
Author: Rainer Jung <rjung@apache.org>
Date:   Mon Nov 2 22:34:45 2009 +0000

    Remove Thread class from jni.
    It does not exist in TC 6 and is neither used
    not especially functional..
    We keep it in TC 5.5 though.
    
    
    git-svn-id: https://svn.apache.org/repos/asf/tomcat/trunk@832165 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/java/org/apache/tomcat/jni/Thread.java b/java/org/apache/tomcat/jni/Thread.java
deleted file mode 100644
index 9990348..0000000
--- a/java/org/apache/tomcat/jni/Thread.java
+++ /dev/null
@@ -1,33 +0,0 @@
-/*
- *  Licensed to the Apache Software Foundation (ASF) under one or more
- *  contributor license agreements.  See the NOTICE file distributed with
- *  this work for additional information regarding copyright ownership.
- *  The ASF licenses this file to You under the Apache License, Version 2.0
- *  (the "License"); you may not use this file except in compliance with
- *  the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-
-package org.apache.tomcat.jni;
-
-/** Thread
- *
- * @author Mladen Turk
- * @version $Revision$, $Date$
- */
-
-public class Thread {
-    
-    /**
-     * Get the current thread ID handle.
-     */
-    public static native long current();    
-
-}
